src/events.js

```diff
@@ -0,0 +1,14 @@
+const { readdirSync } = require("fs");
+require('colors');
```

Comment:
why is this just required but not assigned to a variable

Reply:
Cause It's an extend to String
I can use console.log(String.green)